packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/servers/server/server.routing.js

```diff
@@ -46,6 +45,14 @@ export default /* @ngInject */ ($stateProvider) => {
         Server.getSelected(serverName).then(
           (swsResponse) => new DedicatedServer(swsResponse),
         ),
+      serverType: (server) => {
+        if (server.iam.tags) {
+          return typeof server.iam.tags['ovh:cluster'] !== 'undefined'
+            ? 'node'
+            : 'server';
+        }
+        return 'server';
+      },
```

Comment:
I don't know if it's the good way because i think we have 2 case. So i think we can do it with an bolean. isNode true or false.

Reply:
This was already been in this way in the main PR. And its good way may be in future if have third case it's easy to handle the situation.